functions/backup.bash

```diff
@@ -44,12 +44,12 @@ restore_openhab_config() {
   fi
 
   local backupList
-  local backupPath="/var/lib/openhab2/backups"
+  local backupPath="/var/lib/openhab/backups"
   local filePath
   local fileSelect
   local introText="This will restore a backup of your openHAB configuration using openHAB's builtin backup tool.\\n\\nWould you like to continue?"
 
-  readarray -t backupList < <(ls -alh "${backupPath}"/openhab2-backup-* 2> /dev/null | head -20 | awk -F ' ' '{ print $9 " " $5 }' | xargs -d '\n' -L1 basename | awk -F ' ' '{ print $1 "\n" $1 " " $2 }')
+  readarray -t backupList < <(ls -alh "${backupPath}"/openhab-backup-* 2> /dev/null | head -20 | awk -F ' ' '{ print $9 " " $5 }' | xargs -d '\n' -L1 basename | awk -F ' ' '{ print $1 "\n" $1 " " $2 }')
```

Comment:
this filename is generated by openhab-cli isn't it ?
did you check it's also openhab (no "2") ?

Reply:
Yes, however it wont find any openhab 2 backups anymore.

Reply:
I think it retrieves the path from /etc/default/openhab

Reply:
that's ok